src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -16,6 +18,22 @@ public Rectangle(Integer width, Integer length, String color) {
         this.color = color;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this != o && getClass() != o.getClass()) {
```

Comment:
what about o == null?

Reply:
I tried to do this but InteliJ told me "this != o" covered "o == null".  to add anyway?